services/src/error.rs

```diff
@@ -66,6 +67,13 @@ pub enum Error {
 
     InvalidNamespace,
 
+    InvalidSpatialReference,
+    #[snafu(display("SpatialReferenceMissmatch {} != {}", a, b))]
+    SpatialReferenceMissmatch {
+        a: SpatialReferenceOption,
+        b: SpatialReferenceOption,
+    },
+
```

Comment:
You could maybe also rename the error.

And I would say `expected: {}, found: {}` or does the order not matter for the error and the user?

Reply:
done

Reply:
29a9e1e